content/en/docs/tasks/administer-cluster/running-cloud-controller.md

```diff
@@ -77,18 +74,19 @@ Cloud controller manager does not implement any of the volume controllers found
 
 ### Scalability
 
-In the previous architecture for cloud providers, we relied on kubelets using a local metadata service to retrieve node information about itself. With this new architecture, we now fully rely on the cloud controller managers to retrieve information for all nodes. For very large clusters, you should consider possible bottle necks such as resource requirements and API rate limiting.
+The cloud controller queries your cloud provider's APIs to retrieve information for all nodes. For very large clusters, you should consider possible bottle neckssuch as resource requirements and API rate limiting.
```

Comment:
nit: Line 77, cloud-controller-manager?
For very large clusters, consider possible bottlenecks such as resource requirements and API rate limiting.

Reply:
Ah, “neckssuch” is a typo too.